website/content/docs/configuration/server_join.mdx

```diff
@@ -113,24 +129,43 @@ unless configured otherwise:
 nomad-01.company.local => nomad-01.company.local:4648
 ```
 
-### Via the go-discover interface
+### `go-discover` format
 
 As of Nomad 0.8.4, `retry_join` accepts a unified interface using the
```

Comment:
I have somewhere on a TODO list to remove all these ancient "as of Nomad 0.8..." references :grinning: 

```suggestion
`retry_join` accepts a unified interface using the
```